### PR TITLE
feat: index nature juridique as keyword

### PR DESCRIPTION
**Pull request by HAEKADI**
related to https://github.com/etalab/annuaire-entreprises-search-api/issues/172
Change `nature_juridique_unite_legale` mapping to keyword in order to use a prefix query to filter on it.

### CODE REVIEW

**Review by geoffreyaldebert — APPROVED** [review]
Waw I did not see that we missed that before

**Review by XavierJp — APPROVED** [review]

**Comment by HAEKADI** [comment]
This PR adds `est_service_public` to elasticsearch`s mapping and preprocesses its value based on ` nature_juridique_unite_legale`.

File: data_enrichment.py [hunk 1]
```diff
@@ -67,6 +67,16 @@ def is_entrepreneur_individuel(nature_juridique_unite_legale):
         return False
 
 
+# Service public
+def is_service_public(nature_juridique_unite_legale):
+    if nature_juridique_unite_legale and nature_juridique_unite_legale.startswith(
```
Comment (XavierJp) [inline data_enrichment.py:72]: Ajouter la condition debut de siren = 1 ou 2

**Review by XavierJp — CHANGES_REQUESTED** [review]

File: data_enrichment.py (hunk 1 above)
Comment (HAEKADI) [reply data_enrichment.py]: Done!

File: data_enrichment.py [hunk 2]
```diff
@@ -67,6 +67,19 @@ def is_entrepreneur_individuel(nature_juridique_unite_legale):
         return False
 
 
+# Service public
+def is_service_public(nature_juridique_unite_legale, siren):
+    if (
+        nature_juridique_unite_legale
+        and nature_juridique_unite_legale.startswith(
+            ("3210", "3110", "4", "71", "72", "73", "74")
```
Comment (XavierJp) [inline data_enrichment.py:75]: Nice de pouvoir passer un tuple dans `startswith` !

**Review by XavierJp — APPROVED** [review]

**Pull request merged**